EntityManager/src/test/java/eu/neclab/ngsildbroker/entityhandler/controller/EntityBatchControllerTest.java

```diff
@@ -0,0 +1,334 @@
+	                .when()
+	                .post("/ngsi-ld/v1/entityOperations/upsert")
+	                .then()
+	                   .statusCode(Status.NO_CONTENT.getStatusCode())
+	                   .statusCode(204).extract();
+	            int statusCode = response.statusCode();
+	            assertEquals(204, statusCode);
+	            
+	    }catch(Exception e) {
+	        e.printStackTrace();
+	    }
+
+	}
+
+	/**
+	 * this method is validate the bad request if upsert the multiple entity but
+	 * some entity request is not valid
+	 */
+
+	@Test
+	@Order(6)
+	public void upsertMultipleEntityBadRequestTest() {
+		
+		try {    
+	        ExtractableResponse<Response> response = RestAssured.given()
+	                  .body(badRequestPayload1)
+	                .header(HttpHeaders.CONTENT_TYPE, AppConstants.NGB_APPLICATION_JSON)
+	                 .header(HttpHeaders.ACCEPT,AppConstants.NGB_APPLICATION_JSONLD)
+	                .when()
+	                .post("/ngsi-ld/v1/entityOperations/upsert")
+	                .then()
+	                   .statusCode(Status.BAD_REQUEST.getStatusCode())
+	                   .statusCode(400).extract();
+	            int statusCode = response.statusCode();
+	            assertEquals(400, statusCode);
+	            
+	    }catch(Exception e) {
+	        e.printStackTrace();
+	    }
+
+	}
+
+	/**
+	 * this method is use for delete the multiple entity
+	 */
+	@Test
+	@Order(7)
+	public void deleteMultipleEntityTest() {
+		
+		try {    
+	        ExtractableResponse<Response> response = RestAssured.given()
+	                  .body(deletePayload)
+	                .header(HttpHeaders.CONTENT_TYPE, AppConstants.NGB_APPLICATION_JSON)
+	                 .header(HttpHeaders.ACCEPT,AppConstants.NGB_APPLICATION_JSONLD)
+	                .when()
+	                .post("/ngsi-ld/v1/entityOperations/delete")
+	                .then()
+	                   .statusCode(Status.NO_CONTENT.getStatusCode())
+	                   .statusCode(204).extract();
+	            int statusCode = response.statusCode();
+	            assertEquals(204, statusCode);
+	            
+	    }catch(Exception e) {
+	        e.printStackTrace();
+	    }
+
+	}
+
+	/**
+	 * this method is use for delete the multiple entity but someone entity not
+	 * exist.
+	 */
+	@Test
+	@Order(8)
+	public void deleteMultipleEntityBadRequestTest() {
+		
+		try {    
+	        ExtractableResponse<Response> response = RestAssured.given()
+	                  .body(BadRequestDeletePayload)
+	                .header(HttpHeaders.CONTENT_TYPE, AppConstants.NGB_APPLICATION_JSON)
+	                 .header(HttpHeaders.ACCEPT,AppConstants.NGB_APPLICATION_JSONLD)
+	                .when()
+	                .post("/ngsi-ld/v1/entityOperations/delete")
+	                .then()
+	                   .statusCode(Status.BAD_REQUEST.getStatusCode())
+	                   .statusCode(400).extract();
+	            int statusCode = response.statusCode();
```

Comment:
Same as above. please this changes  everywhere.

Reply:
done